comms/src/connection_manager/tests/manager.rs

```diff
@@ -149,6 +149,13 @@ async fn simultaneous_dial_events() {
     result1.unwrap();
     result2.unwrap();
 
+    // Wait for listening and peer connected events
```

Comment:
Fix for flakey test - wait until peer definitely has connected (i.e. after tie breaking) before shutting down